pages/common/yacas.md

```diff
@@ -0,0 +1,28 @@
+# yacas
+
+> Yet Another Computer Algebra System.
+> More information: <http://www.yacas.org>.
+
+- Start an interactive console session:
+
+`yacas`
+
+- In the console session, execute a statement:
+
+`In> {{Integrate(x)Cos(x);}}`
+
+- In the console session, display an example:
+
+`In> {{Example();}}`
+
+- Quit from the console session:
```

Comment:
```suggestion
- Quit `yacas`:
```

Reply:
accepted this suggestion and pushed an update.

Reply:
I take that back. I actually used "the `yacas` session" like the other examples.